framework/src/st_things/st_things.c

```diff
@@ -37,6 +37,11 @@
 
 #define TAG "[st_things_sdk]"
 
+#define SEC_ATRRIBUTE_PUSH_MESG_MSGID		"x.org.iotivity.ns.messageid"
```

Comment:
Could you please use the spaces instead of tabs. also.

Reply:
fixed.